packages/fuels/tests/scripts/script_raw_slice/.gitignore

```diff
@@ -0,0 +1,2 @@
+out
```

Comment:
We should probably see what to do about the `.gitignores` that seem to have started being generated by forc.

Currently only this forc project and one regarding predicate data commit a `.gitignore`. We can probably lift that out somewhere on the level of the whole e2e dir.

Reply:
I added the `.gitignore` pattern for test projects in the root gitignore, the same way we ignore build artifacts.